### PR TITLE
ui: Add more explanatory texts for empty states

### PR DESCRIPTION
**Pull request by johncowen**
Related: https://github.com/hashicorp/consul/issues/10426#issuecomment-868432159

This PR does two things:

1. From a human perspective it updates all our 'empty state' texts to include information on the [ACL policy dispositions](https://www.consul.io/docs/security/acl/acl-rules#policy-dispositions) required to view data on that specific view.
2. Moves all of this related text to our translation files.

Small note here:

I noticed whilst doing this that we seem to be very close to full i18n'ed text in all our Routes now. In fact we are probably at a stage where we should make a move to specifically go ahead and make sure all our Routes are 100% i18n'ed seeing as we must be about 95% there, we may aswell go ahead and get the last 5% done soon-ish.

### CODE REVIEW

File: ui/packages/consul-ui/app/templates/dc/services/instance/upstreams.hbs
Comment (natmegs) [inline ui/packages/consul-ui/app/templates/dc/services/instance/upstreams.hbs:59]: I think these two might need `htmlSafe=true`?

File: ui/packages/consul-ui/app/templates/dc/services/instance/exposedpaths.hbs
Comment (natmegs) [inline ui/packages/consul-ui/app/templates/dc/services/instance/exposedpaths.hbs:10]: Do we need to pass the `CONSUL_DOCS_URL` variable to the translations?

Comment (johncowen) [reply ui/packages/consul-ui/app/templates/dc/services/instance/exposedpaths.hbs:10]: Ah good question! Nah, they are all auto injected here:

https://github.com/hashicorp/consul/blob/9b22300c59c2751e08fd339fc77e1af2f2d7dcf4/ui/packages/consul-ui/app/services/i18n.js#L15-L23

**Comment by johncowen** [comment]
This has a little conflict now, likely to be done after the weekend will ping properly once its ready again

**Comment by johncowen** [comment]
😬 YAML conflicts are far more hairy than they should be. I have a few improvements in the works that I've not been able to PR as yet for various reasons that would help here. Pretty sure this one is correct now, looking at the diff here there was a bit of moving around of yaml-ness.

@natmegs I guess it's ready for review now, but I'll be holding off on the merge for a little while whichever.

**Review by evrowe — APPROVED** [review]

**Comment by johncowen** [comment]
🎉 Thanks for the approval!

Just a quick note here, there are a couple of questions outstanding on the textual content here so just gonna leave this hanging a little more until we get clarity on that. We may just merge and follow up with an additional PR or just pop more on that end here. Following up offline.

**Review by natmegs — APPROVED** [review]

**Comment by johncowen** [comment]
Just to follow up here, I noticed the test we have added here might not be correct, if I've understood correctly the text we are using here only specifies that you need `service:read` for the service listing page, whereas according to the user below you need a few more policy dispositions in order to see that view.

https://github.com/hashicorp/consul/issues/10426#issuecomment-868414378

> It turns out that the anonymous token needs to also have read access to "node" and/or "agent". Should have been obvious that it needs it since it displays info about the nodes where the service is registered 🙄

So I've checked in with the team to see whether to merge this or wait until we get corrected text (if it does need correcting)

**Comment by johncowen** [comment]
I tried this out and `node:read` is required for viewing the service listing page, so I added that piece of text. Another rebase was required which needed a little bit of manual yaml'ing also, so that all went in the same commit.

Let me know if anything else needs to happen here, otherwise I'll leave it hanging for a little while longer then merge.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/632517.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit e62275642b3dd9a7a64f465fdb117a2b0a1c763f onto `release/1.12.x` succeeded!

**Pull request merged**
